File: MSMA_Backend/src/main/java/com/spring/dto/response/GenreResponse.java
package com.spring.dto.response;

import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder(toBuilder = true)
public class GenreResponse {
    private Long id;
    private String name;
    private String imageUrl;
    private String briefDescription;
    private String fullDescription;
    private String createdDate;
    private String lastModifiedDate;
}

File: MSMA/app/src/main/java/vn/edu/usth/msma/data/dto/response/management/GenreResponse.kt
package vn.edu.usth.msma.data.dto.response.management

import com.google.gson.annotations.SerializedName

data class GenreResponse(
    @SerializedName("id")
    val id: Long,

    @SerializedName("name")
    val name: String,

    @SerializedName("imageUrl")
    val imageUrl: String,

    @SerializedName("briefDescription")
    val briefDescription: String,

    @SerializedName("fullDescription")
    val fullDescription: String,

    @SerializedName("createdDate")
    val createdDate: String,

    @SerializedName("lastModifiedDate")
    val lastModifiedDate: String
)